Update logicals.qmd (#1166)
hadley/r4ds: logicals.qmd

When you use a logical vector in a numeric context, `TRUE` becomes 1 and `FALSE` becomes 0.
This makes `sum()` and `mean()` very useful with logical vectors because `sum(x)` will give the number of `TRUE`s and `mean(x)` the proportion of `TRUE`s.

> Let's see the distribution of the proportion of delayed flights for each day of the year as shown in @fig-prop-delayed-dist.

Author: Please ignore my change to "That lets us". For some reason, I read it multiple times in my mind as "Then lets us... ". My apologies!

Reviewer: ```suggestion
That lets us see the distribution of delays across the days of the year as shown in @fig-prop-delayed-dist
```
